src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -299,18 +300,29 @@ public static void setParam(final String[] args, final String confFileName) {
         List<String> localwitness = config.getStringList("localwitnesskeystore");
         if (localwitness.size() > 0) {
           String fileName = System.getProperty("user.dir") + "/" + localwitness.get(0);
-          System.out.println("Please input your password.");
-          String password = WalletUtils.inputPassword();
+          String password;
+          if (StringUtils.isEmpty(INSTANCE.password)) {
+            System.out.println("Please input your password.");
+            password = WalletUtils.inputPassword();
+          } else {
+            password = INSTANCE.password;
+            INSTANCE.password = "";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'password' detected in this expression, review this potentially hardcoded credential. ([squid:S2068](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=950&issues=AWPTFGGfA0k93xG_IQNa&open=AWPTFGGfA0k93xG_IQNa)

[ISSUE_KEY]: AWPTFGGfA0k93xG_IQNa